doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,65 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication.
+
+//Are all steps in the appliance console menu?
```

Comment:
I don't think so. I think the user will need to edit `database.yml` manually and will also need to stop the database service on the standby by hand.